Dockerfile

```diff
@@ -1,14 +1,12 @@
 FROM molecularplayground/node-js
-RUN apk add --no-cache bash
+RUN apk add bash
```

Comment:
Got it. Thanks.

On Wed, Jun 1, 2016 at 9:40 AM, Cheyan Setayesh <notifications@github.com>
wrote:

> In Dockerfile
> <https://github.com/ripples/lv-server/pull/1#discussion_r65361225>:
>
> > @@ -1,14 +1,12 @@
> >  FROM molecularplayground/node-js
> > -RUN apk add --no-cache bash
> > +RUN apk add bash
>
> The apk package manager could use cache to fetch bash, I got rid of no
> cache so we could use this, but I was mistakened to think this would work
> cause Ryan informed me that the images are built from scratch. I got rid of
> it the line anyways cause we already had sh installed.
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/ripples/lv-server/pull/1/files/3ab96d037e0fd9c49f763d4cd21427b50e488598#r65361225>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAVQKlEjpuKrQmYPeNeBZpW8vy4ezA3Hks5qHYvngaJpZM4IphkJ>
> .
>